File: java/PML/src/tags/TagIf.java
package tags;

import parser.PMLParser;

public class TagIf extends Tag{

	@Override
	public void output() {
		String cond = getAttributes().get("cond");
		
		System.out.println("<?php if (" + cond + ") : ?>");
		PMLParser.parse(this.getChildTags());
		System.out.println("<?php endif; ?>");
	}
}

File: php/classes/PML/Tags/TagIf.php
<?php namespace PML\Tags;

use PML\Parser\PMLParser;

class TagIf extends Tag {

    public function output()
    {
        $atts = $this->getAttributes();
        $cond = $atts['cond'];
        echo '<?php if(' . $cond . ') : ?>';
        PMLParser::parse($this->getChildTags());
        echo '<?php endif; ?>';
    }
}
